### DISCUSSION TITLE
SEO

### DISCUSSION THREAD

**Discussion by hirockstar** [spencerwooo/onedrive-vercel-index#590] [Ideas]
Can you add SEO option in this project.

**Comment by spencerwooo** [comment]
SEO is a big word, too big for this project. Won't consider in the near future.

**Reply by hirockstar** [reply]
ahh okey it will be good for me it will possible in near future update.

**Comment by ksiegowakrakow** [comment]
Has something changed in adapting to seo?

**Reply by myl7** [reply]
No. Since the project relies on Next.js, to do SEO at least all pages should be SSRed, which requires a big refactoring for all code and seems not worthy.

**Comment by hirockstar** [comment]
@ksiegowakrakow we will work on it soon let's see if its work for us.
